LeadKit/LeadKit/Extensions/Observable/Observable+ToastErrorLogging.swift

```diff
@@ -29,11 +29,12 @@ public extension Observable {
     ///
     /// - Returns: The source sequence with the side-effecting behavior applied.
     func showErrorsInToastInDebugMode() -> Observable<Observable.E> {
-        return `do`(onError: { (error) in
-            #if DEBUG
-                UIApplication.shared.keyWindow?.makeToast(error.localizedDescription)
-            #endif
-        })
+        return observeOn(MainScheduler.instance)
+            .do(onError: { (error) in
+                #if DEBUG
```

Comment:
And I think that observeOn(MainScheduler.instance) is not good idea.

How about this solution?
dispatch_async(GlobalMainQueue) {
        UIApplication.shared.keyWindow?.makeToast(error.localizedDescription)
}